modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -869,18 +869,32 @@ void GDScriptAnalyzer::resolve_class_body(GDScriptParser::ClassNode *p_class) {
 	GDScriptParser::ClassNode *previous_class = parser->current_class;
 	parser->current_class = p_class;
 
-	// Do functions now.
+	// Do functions and properties now.
 	for (int i = 0; i < p_class->members.size(); i++) {
 		GDScriptParser::ClassNode::Member member = p_class->members[i];
-		if (member.type != GDScriptParser::ClassNode::Member::FUNCTION) {
-			continue;
-		}
+		if (member.type == GDScriptParser::ClassNode::Member::FUNCTION) {
+			resolve_function_body(member.function);
 
-		resolve_function_body(member.function);
+			// Apply annotations.
+			for (GDScriptParser::AnnotationNode *&E : member.function->annotations) {
+				E->apply(parser, member.function);
+			}
+		} else if (member.type == GDScriptParser::ClassNode::Member::VARIABLE && member.variable->property != GDScriptParser::VariableNode::PropertyStyle::PROP_NONE) {
```

Comment:
We usually don't add the enum name when referring to its elements. Although this is more of a legacy convention since older compilers didn't support it, given that in other places this code doesn't have the enum name (e.g. in `GDScriptParser::ClassNode::Member::FUNCTION` it doesn't specify the `Type` enum), I think it should be better to keep the current convention, so this should use `GDScriptParser::VariableNode::PROP_NONE`.